custom-elements/CustomElementRegistry-constructor-and-callbacks-are-held-strongly.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CustomElementInterface holds constructors and callbacks strongly, preventing them from being GCed if there are no other references</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/custom-elements.html#concept-custom-element-definition-lifecycle-callbacks">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/garbage-collect.js"></script>
```

Comment:
Don't we also need to add this file?

Reply:
It came in https://github.com/web-platform-tests/wpt/pull/35386.

Would be nice to have it globally though, as there 4-5 copies of pretty much the same helper across the WPT.

_cc_ @gsnedders

Reply:
Hm... ok. Perhaps we can move it to /resources/ at the top. But that's probably a follow up.

Reply:
See #7899 re: GC triggers generally; `/common` would be better than `/resources` (I don't think we really document the distinction well, but essentially `/resources` is treated as WPT infra stuff, whereas `/common` is a free-for-all).